src/net.h

```diff
@@ -252,7 +252,7 @@ class CConnman
         m_onion_binds = connOptions.onion_binds;
     }
 
-    CConnman(uint64_t seed0, uint64_t seed1, bool network_active = true);
```

Comment:
Look like the fuzzer is complaining about this and needs to be updated to take this additional arg type:

```
test/fuzz/connman.cpp:26:14: error: no matching constructor for initialization of 'CConnman'
    CConnman connman{fuzzed_data_provider.ConsumeIntegral<uint64_t>(), fuzzed_data_provider.ConsumeIntegral<uint64_t>(), fuzzed_data_provider.ConsumeBool()};
             ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./net.h:255:5: note: candidate constructor not viable: no known conversion from 'bool' to 'CAddrMan &' for 3rd argument
    CConnman(uint64_t seed0, uint64_t seed1, CAddrMan& addrman, bool network_active = true);
    ^
./net.h:187:7: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 3 were provided
class CConnman
      ^
1 error generated.
```

Reply:
Thanks! Should be fixed now.